### DISCUSSION TITLE
Duplicate project [May 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5652] [Discord Threads]
> 🙋‍♂️ **j0**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Currently no. Work in progress on this feature. We should have it enabled with in next few release.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1093975620398088273)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
